packages/notifications/src/inAppMessaging/providers/pinpoint/utils/constants.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+export const PINPOINT_KEY_PREFIX = 'AWSPinpoint';
+export const INAPP_STORAGE_KEY_SUFFIX = '_inAppMessages';
+export const UPDATEENDPOINT_CATEGORY = 'InAppMessaging';
```

Comment:
Nit: Since everything is scoped to in app messaging already
```suggestion
export const STORAGE_KEY_SUFFIX = '_inAppMessages';
export const CATEGORY = 'InAppMessaging';
```